sysboundary/sysboundary.cpp

```diff
@@ -431,46 +283,43 @@ bool belongsToLayer(const int layer, const int x, const int y, const int z,
    return belongs;
 }
 
-/*!\brief Classify all simulation cells with respect to the system boundary conditions.
- *
- * Loops through all cells and and for each assigns the correct sysBoundaryFlag depending on
- * the return value of each SysBoundaryCondition's assignSysBoundary.
+/*!\brief Classify all simulation cells with respect to the boundary conditions.
  *
+ * Loops through all cells and and for each assigns the correct sysBoundaryFlag
+ * depending on the return value of each SysBoundaryCondition's assignBoundary.
  * \param mpiGrid Grid
  */
-bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell, dccrg::Cartesian_Geometry>& mpiGrid,
+void SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell, dccrg::Cartesian_Geometry>& mpiGrid,
                                 FsGrid<fsgrids::technical, FS_STENCIL_WIDTH>& technicalGrid) {
-   bool success = true;
-   const vector<CellID>& cells = getLocalCells();
+   vector<CellID> cells = mpiGrid.get_cells();
    auto localSize = technicalGrid.getLocalSize().data();
 
-   /*set all cells to default value, not_sysboundary*/
-   for (uint i = 0; i < cells.size(); i++) {
+   // Set all cells to default value
+   for (uint i = 0; i < cells.size(); i++)
```

Comment:
Again, I find that for-loops would be better with the curly braces, but it's a matter of preference.
And in matters of preference, I recommend to err on the side of minimazing changes.